src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -282,11 +282,22 @@ func (gen *PythonStartupScriptGenerator) buildGunicornCommandForModule(module st
 	}
 	
 	pythonEnableGunicornMultiWorkers := common.GetBooleanEnvironmentVariable(consts.PythonEnableGunicornMultiWorkersEnvVarName)
+
 	if pythonEnableGunicornMultiWorkers {
 		// 2N+1 number of workers is recommended by Gunicorn docs.
 		// Where N is the number of CPU threads.
 		// One worker will be reading or writing from the socket while the other worker is processing a request.
-		workers := strconv.Itoa((2 * runtime.NumCPU()) + 1)
+		// For MWMT (Multi Worker Multi Thread), user specifies two environment variables to enable MWMT.
+		// Otherwise, this script will use the recommended setting by Gunicorn.
+		pythonCustomWorkerNum := os.Getenv(consts.PythonGunicornCustomWorkerNum)
+		pythonCustomThreadNum := os.Getenv(consts.PythonGunicornCustomThreadNum)
+		workers := ""
+		if ((pythonCustomWorkerNum != "") && (pythonCustomThreadNum != "")) {
+			workers = pythonCustomWorkerNum
+			args = appendArgs(args, "--threads="+pythonCustomThreadNum)
+		} else {
+			workers = strconv.Itoa((2 * runtime.NumCPU()) + 1)
+		}
```

Comment:
This handles the case that Antares requested (_i.e.,_ setting both values based on the SKU), but from Oryx's point of view, we should be able to handle users who want to provide different combination of gunicorn arguments when running their application. The logic above ignores the cases where the user provides exactly one of the worker or threads arguments, so we should clean it up a little to account for either provided.

```go
if (pythonCustomThreadNum != "") {
    args = appendArgs(args, "--threads="+pythonCustomThreadNum)
}

workers := strconv.Itoa((2 * runtime.NumCPU()) + 1)
if (pythonCustomWorkerNum != "") {
    workers = pythonCustomWorkerNum
}

args = appendArgs(args, "--workers="+workers)
```

Reply:
That's a good point. I haven't thought about that corner case. Just added it. I've also added some documentations in configuration.md